Zend/zend_fibers.h

```diff
@@ -25,6 +25,15 @@
 
 BEGIN_EXTERN_C()
 
+#define ZEND_FIBER_C_STACK_ALIGNMENT (4 * 1024)
+#define ZEND_FIBER_DEFAULT_C_STACK_SIZE (ZEND_FIBER_C_STACK_ALIGNMENT * (sizeof(void *) * 64))
+#define ZEND_FIBER_MIN_C_STACK_SIZE (128 * 1024)
```

Comment:
Why this as a minimum stack size. `ZEND_FIBER_C_STACK_ALIGNMENT * (ZEND_FIBER_GUARD_PAGES + 1)` should work, so why require more if the user wishes to set it to the minimum?

Reply:
After we rely on the virtual memory mechanism, too small a C stack size does not seem to make any sense, it will only increase the risk of the crash.
Professionals can still remove the limitation by changing the macro definition.